SekoiaIO/endpoint/_meta/smart-descriptions.json

```diff
@@ -11,6 +11,17 @@
       }
     ]
   },
+  {
+    "value": "Action: {event.action} - Executable {process.executable} - Hostname: {host.hostname}",
+    "conditions": [
+      {
+        "field": "event.action"
+      },
+      {
+        "field": "process.executable"
+      }
+    ]
+  },
```

Comment:
If this description is generic, could you move it to the end of the list?
The algorithm will always select the first matching smart descriptions, so this smart description could override more relevant smart descriptions.

Reply:
Done :)